common/src/main/scala/auditor/package.scala

```diff
@@ -1,26 +1,63 @@
 import java.net.URL
 
-import models.Topic
+import models.{Topic, TopicTypes}
+import play.api.Logger
 import play.api.libs.json.Json
+import play.api.libs.ws.WSClient
 
 import scala.concurrent.{ExecutionContext, Future}
+import scala.util.Try
 
 package object auditor {
 
-  case class AuditorGroupConfig(hosts: Set[String])
+  case class ContentApiConfig(url: String, apiKey: String)
 
-  case class Auditor(host: URL)
+  case class AuditorGroupConfig(hosts: Set[String], contentApiConfig: ContentApiConfig)
+
+  sealed trait Auditor {
+    def expiredTopics(topics: Set[Topic])(implicit ec: ExecutionContext): Future[Set[Topic]]
+  }
+
+  case class LiveblogAuditor(wsClient: WSClient, config: ContentApiConfig) extends Auditor {
+
+    override def expiredTopics(topics: Set[Topic])(implicit ec: ExecutionContext): Future[Set[Topic]] = {
+      val contentTopics = topics.filter(_.`type` == TopicTypes.Content)
+
+      Future.traverse(contentTopics) { topic =>
+        isLiveBloggingNow(topic.name).map {
+          case true => None
+          case false => Some(topic)
+        }
+      }.map(_.flatten)
+    }
+
+    def isLiveBloggingNow(id: String)(implicit ec: ExecutionContext): Future[Boolean] = {
+      val url = s"${config.url}/$id?show-fields=liveBloggingNow&api-key=${config.apiKey}"
+      wsClient.url(url).get().map { response =>
+        val liveBloggingNowField = response.json \ "response" \ "content" \ "fields" \ "liveBloggingNow"
+        liveBloggingNowField.validate[String].asOpt.flatMap(s => Try(s.toBoolean).toOption).getOrElse(false)
+      } recover { case _ => true }
```

Comment:
why defaulting to false above, but recovering to true here, I feel like I missed something?